services/API-service/src/api/data/data.controller.ts

```diff
@@ -67,16 +68,16 @@ export class DataController {
   }
 
   @ApiOperation({ summary: 'Get triggered areas' })
-  @ApiParam({ name: 'event', required: true, type: 'number' })
-  @Get('triggered-areas/:event')
+  @ApiParam({ name: 'countryCode', required: true, type: 'string' })
+  @Get('triggered-areas/:countryCode')
   public async getTriggeredAreas(@Param() params): Promise<TriggeredArea[]> {
-    return await this.dataService.getTriggeredAreas(params.event);
+    return await this.dataService.getTriggeredAreas(params.countryCode);
   }
 
-  @ApiOperation({ summary: 'Get active event' })
+  @ApiOperation({ summary: 'Get active country event' })
```

Comment:
I am not sure about this. Since this controller returns a sort of aggegrate of all pcode events in a country. So it just returns:

```
  public country_code: string;
  public start_date: string;
  public end_date: string;
  public active_trigger: boolean;
```

for a country in which start_date is the earliest start date of all pcode events and end_date the latest end_date of all pcode event. I think the name CountryEvent is sort of okay? Or maybe AggregateCountryEvent?

Reply:
oh, you're right. I interpreted this as something that returns a list. `AggregateCountryEvent` sounds good but we already have something we refer to as `aggregate` so let's call it `getEventSummaryForCountry`?